### PR TITLE
Us130328/jtran add quizzing attempt dropdown

### PR DESCRIPTION
**Pull request by JamieTran**
[US130328](https://rally1.rallydev.com/#/42960320374d/custom/236803223728?detail=%2Fuserstory%2F605343765553)

Added the visual web component for the quizzing attempts dropdown
- Displays the attempts and the complete summary in the dropdown
- Is not hooked up to the left-hand panel to display the selection
image

### CODE REVIEW

File: test/perceptual/components/header/consistent-evaluation-lcb.visual-diff.html [hunk 1]
```diff
@@ -103,5 +129,15 @@
 			</div>
 		</div>
 
+		<div id="is-quiz-attempts">
```
Comment (JamieTran) [inline test/perceptual/components/header/consistent-evaluation-lcb.visual-diff.html:132]: I followed the pattern of adding the visual diff test to LCB like the submission dropdown, but if we think that it's important to have their own tests then I can make it happen.

File: components/header/consistent-evaluation-attempt-context.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+			<select class="d2l-input-select d2l-truncate" @change=${this._onAttemptChange}>
+				${Object.values(this.quizAttemptInfoList).map(attempt => html`
+					<option value=${attempt.attemptNumber} label=${this.localize('quizAttempt', { num: attempt.attemptNumber })} ></option>
+				`)}
+				<option value="-1" label=${this.localize('completionSummary')} ></option>
+			</select>
+		`;
+	}
+
+	_onAttemptChange() {
```
Comment (JamieTran) [inline components/header/consistent-evaluation-attempt-context.js:63]: This is encapsulated in [US130328](https://rally1.rallydev.com/#/42960320374d/custom/236803223728?detail=%2Fuserstory%2F605343765553)

File: components/consistent-evaluation-page.js [hunk 3]
```diff
@@ -860,6 +860,7 @@ export default class ConsistentEvaluationPage extends SkeletonMixin(LocalizeCons
 					.currentFileId=${this.currentFileId}
 					.submissionInfo=${this.submissionInfo}
 					.groupInfo=${this.groupInfo}
+					.quizAttemptInfoList=${this.quizAttemptInfoList}
```
Comment (JamieTran) [inline components/consistent-evaluation-page.js:863]: With talks about a refactor, where we get this information might change but the component shouldn't

Comment (maddiegabriel) [reply components/consistent-evaluation-page.js:863]: makes sense 👍

File: test/perceptual/components/header/consistent-evaluation-lcb.visual-diff.html (hunk 1 above)
Comment (maddiegabriel) [reply test/perceptual/components/header/consistent-evaluation-lcb.visual-diff.html:132]: IMO the way you have it is fine 👍

**Review by maddiegabriel — APPROVED** [review]
lookin' good JT! 👍

**Review by tdevos-d2l — APPROVED** [review]

**Pull request merged**